### PR TITLE
Fix unread comment icon not shown for the sharee

### LINKED ISSUE #12027: The icon of new comment in the share is missing
Hi,
Could you add the icon of new comments in the shares? 
Also the icon is missing anywhere at the recipient if the folder was shared.
1. Add a recipient to the shares
2. To comment from the sender. The sender is have, the recipient is not.
image

### PR DESCRIPTION
**Pull request by rullzer**
Fix #12027

### CODE REVIEW

File: apps/dav/lib/Connector/Sabre/CommentPropertiesPlugin.php [hunk 1]
```diff
@@ -112,11 +112,11 @@ public function handleGetProperties(
...
 					$parentPath = '/';
 				}
-				// if we already cached the folder this file is in we know there are no shares for this file
+				// if we already cached the folder this file is in we know there are no comments for this file
 				if (array_search($parentPath, $this->cachedFolders) === false) {
-					return $this->getUnreadCount($node);
-				} else {
 					return 0;
+				} else {
+					return $this->getUnreadCount($node);
```
Comment (skjnldsv) [inline apps/dav/lib/Connector/Sabre/CommentPropertiesPlugin.php]: :tada:

**Review by skjnldsv — APPROVED** [review]

**Comment by skjnldsv** [comment]
Timeout in apps unrelated!
`Disable button in the app list for Auditing / Logging could not be found after 100 seconds (NoSuchElementException)`

**Review by blizzz — APPROVED** [review]

**Pull request merged**